test/README.md

```diff
@@ -0,0 +1,15 @@
+# Docker stacks testing
```

Comment:
There was no README for testing, so I added one. Makes it easy for newcomers to contribute to tests.

Reply:
I love this practice! It makes my review easier and helps any future maintenance or contribution by having in a README file going onwards!

Reply:
Excellent writeup! I think you wrote it on a very suitable level - not too detailed or not too abstract etc.